webapp/public/locale/en/strings.json

```diff
@@ -1,6 +1,5 @@
 {
   "common": {
-    "add_money_to_account": "Press here to add money to your account",
```

Comment:
Why was this removed?

Reply:
It was replaced with an alert component and context with an "Add funds" button.